index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Random Heading</title>
+    <style>
+        body {
+            font-family: Arial, sans-serif;
```

Comment:
use a more exciting font than arial

Reply:
I have updated the font to 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif for a more exciting look.